frontend/src/metabase/parameters/components/ParameterValueWidget.tsx

```diff
@@ -225,11 +228,17 @@ export const ParameterValueWidget = ({
                   size={16}
                 />
               )}
-              <div className={cx(CS.mr1, CS.textNoWrap)}>
+              <div
+                className={cx(CS.mr1)}
+                style={
+                  isStringParameter(parameter) ? { maxWidth: "190px" } : {}
+                }
+              >
```

Comment:
Limiting this to string parameters so we don't accidentally crop something else, e.g. a data filter value:


![CleanShot 2024-07-26 at 15 30 11@2x](https://github.com/user-attachments/assets/99860067-2454-46c5-bec1-eef169343486)